docs/api_walkthrough.rst

```diff
@@ -1,34 +1,29 @@
 Getting started with the Raiden API
 ###################################
 
-
-Contents:
-
 .. toctree::
-   :maxdepth: 2
-
-
+  :maxdepth: 3
 
 Introduction
-*************
-Raiden has a Restful API with URL endpoints corresponding to actions that the user can perform with his channels. The endpoints accept and return JSON encoded objects. The api url path always contains the api version in order to differentiate queries to different API versions. All queries start with: ``/api/<version>/`` where ``<version>`` is an integer representing the current API version.
+=============
+Raiden has a Restful API with URL endpoints corresponding to actions that the user can perform with its channels. The endpoints accept and return JSON encoded objects. The API URL path always contains the API version in order to differentiate queries to different API versions. All queries start with: ``/api/<version>/`` where ``<version>`` is an integer representing the current API version.
```

Comment:
The proper way to be gender neutral is to use plural their and not neutral form.